modules/sharedaddy/sharing.css

```diff
@@ -423,8 +480,8 @@ body .sd-social-icon .sd-content li.share-custom a span {
 
 .sd-social-official .sd-content .sharing-hidden ul>li.share-end {
 	clear: both;
-	margin: 0;
-	height: 0;
+	margin: 0!important;
+	height: 0!important;
```

Comment:
I'm not sure it's a hard and fast rule, but the CSS guidelines example shows a space before `!important` declarations. Also, it might be worth adding a comment why we need `!important` here instead of being more specific.